pkg/nmi/server/server.go

```diff
@@ -125,6 +132,49 @@ func (fn appHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	latency := time.Since(start)
 	logger.Infof("Status (%d) took %d ns", rw.statusCode, latency.Nanoseconds())
 }
+func (s *Server) hostHandler(logger *log.Entry, w http.ResponseWriter, r *http.Request) {
+	hostIP := parseRemoteAddr(r.RemoteAddr)
+	 if hostIP != "127.0.0.1" {
+		msg := "request remote address is not from a host"
+		logger.Error(msg)
+		http.Error(w, msg, http.StatusInternalServerError)
+		return
+	 }
+
+	 podns, podname := parseRequstHeader(r)
+	 if podns == "" || podname == "" {
+		 logger.Errorf("Error getting podname and podns from request")
```

Comment:
dont have to use "error" since the log is already marked as error, some like "missing pod name or namespaces in request header"